src/v2/styles/ChannelPreview/ChannelPreview-layout.scss

```diff
@@ -8,7 +8,7 @@
   width: 100%;
   padding: var(--str-chat__spacing-3) var(--str-chat__spacing-2);
   margin: 0;
-  text-align: start;
+  text-align: flex-start;
```

Comment:
@MartinCupela I think this should stay `start`

Reply:
fixed